nicegui/socket_manager.py

```diff
@@ -0,0 +1,105 @@
+from typing import Union
```

Comment:
What a pity that [fastapi-socketio](https://github.com/pyropy/fastapi-socketio) didn't release any update since December 2021. But this workaround seems valid.

Reply:
I requested a new release, hope package is not dead: 
[https://github.com/pyropy/fastapi-socketio/issues/41](https://github.com/pyropy/fastapi-socketio/issues/41)